src/components/ResourceList/components/FilterControl/tests/FilterControl.test.tsx

```diff
@@ -22,7 +22,7 @@ describe('<FilterControl />', () => {
     selectMode: false,
     resourceName: {
       singular: 'item',
-      plural: 'items,',
+      plural: 'items',
```

Comment:
why did you remove the `s`

Reply:
I didn't remove the `s` it was a `,` that seemed to have no tests against it. So when I was writing my test and checking the default the value was `Search items,` and that made no sense to me.